### PR TITLE
feat(hub): banner ip failover repricing

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          |  `develop` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-9886 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I added a banner for ip failover repricing

## Related
2api feature-availability (hub:bannerIpRepricing)

:flags: Translation:
- [ ] TRANS-45622

### CODE REVIEW

File: packages/manager/apps/hub/src/dashboard/dashboard.service.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default class DashboardService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  ipFailover() {
```
Comment (frenautvh) [inline packages/manager/apps/hub/src/dashboard/dashboard.service.js:7]: a good practice is to use a verb for function name, "ipFailover" doesn't mean much but "listIpFailover" or "getIpsFailover" provides more infos on what the function does

**Review by frenautvh — APPROVED** [review]

File: packages/manager/apps/hub/src/dashboard/dashboard.controller.js [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
 export default class DashboardController {
+  /* @ngInject */
+  constructor(atInternet, hubDashboardService, ovhFeatureFlipping) {
+    this.atInternet = atInternet;
+    this.hubDashboardService = hubDashboardService;
+    this.ovhFeatureFlipping = ovhFeatureFlipping;
+  }
+
   $onInit() {
+    this.ovhFeatureFlipping
```
Comment (antleblanc) [inline packages/manager/apps/hub/src/dashboard/dashboard.controller.js:15]: **question:** Shouldn't we wrap all async calls in a `$q.all` as we are on a `$onInit` hook?

File: packages/manager/apps/hub/src/dashboard/dashboard.controller.js [hunk 3]
```diff
@@ -1,7 +1,43 @@
...
+
+    this.ipFailover();
+
     return this.trackingPrefix.then((prefix) => {
       this.prefix = prefix;
     });
   }
+
+  ipFailover() {
+    this.hubDashboardService.ipFailover().then((ipFailover) => {
```
Comment (antleblanc) [inline packages/manager/apps/hub/src/dashboard/dashboard.controller.js:30]:
```suggestion
    return this.hubDashboardService.ipFailover().then((ipFailover) => {
```

File: packages/manager/apps/hub/src/dashboard/dashboard.html [hunk 4]
```diff
@@ -6,6 +6,29 @@
...
+                <span
+                    data-translate="manager_hub_dashboard_ipFailover_repricing_message"
+                >
+                </span>
+                <!-- The href will be ready in september -->
+                <a
+                    data-translate="manager_hub_dashboard_ipFailover_repricing_link"
+                    data-ng-click="$ctrl.trackClickImpression()"
+                    data-ng-href=""
+                    target="_blank"
```
Comment (antleblanc) [inline packages/manager/apps/hub/src/dashboard/dashboard.html:28]:
```suggestion
                    target="_blank"
                    rel="noopener"
```

File: packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -1,5 +1,8 @@
 {
   "manager_hub_dashboard": "Tableau de bord",
   "manager_hub_dashboard_overview": "Vue d'ensemble de mon activité",
-  "manager_hub_dashboard_services": "Mes produits et services"
+  "manager_hub_dashboard_services": "Mes produits et services",
+  "manager_hub_dashboard_ipFailover_repricing_title": "Évolution des prix des adresses IPv4 additionnelles",
+  "manager_hub_dashboard_ipFailover_repricing_message": "A partir du 6 octobre prochain, une nouvelle tarification entre en vigueur pour les adresses IPv4 additionnelles.",
```
Comment (antleblanc) [inline packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json:6]:
```suggestion
  "manager_hub_dashboard_ipFailover_repricing_message": "À partir du 6 Octobre prochain, une nouvelle tarification entre en vigueur pour les adresses IPv4 additionnelles.",
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/apps/hub/src/dashboard/dashboard.service.js (hunk 1 above)
Comment (ghyenne) [reply packages/manager/apps/hub/src/dashboard/dashboard.service.js]: Its's applied.

File: packages/manager/apps/hub/src/dashboard/dashboard.controller.js (hunk 3 above)
Comment (ghyenne) [reply packages/manager/apps/hub/src/dashboard/dashboard.controller.js]: Its's applied.

File: packages/manager/apps/hub/src/dashboard/dashboard.html (hunk 4 above)
Comment (ghyenne) [reply packages/manager/apps/hub/src/dashboard/dashboard.html:28]: Its's applied.

File: packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json (hunk 5 above)
Comment (ghyenne) [reply packages/manager/apps/hub/src/dashboard/translations/Messages_fr_FR.json]: Its's applied.

File: packages/manager/apps/hub/src/dashboard/dashboard.controller.js (hunk 2 above)
Comment (ghyenne) [reply packages/manager/apps/hub/src/dashboard/dashboard.controller.js]: Its's applied.

**Review by antleblanc — APPROVED** [review]
LGTM - still missing the `href` value.

**Review by anooparveti — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request closed without merging**